src/medium-zoom.js

```diff
@@ -390,6 +390,16 @@ const mediumZoom = (
     target.zoomedHd && (target.zoomedHd.style.transform = transform)
   }
 
+  const ZoomEvent = (event, params = { bubbles: false, cancelable: false, detail: undefined }) => {
+    if (typeof window.CustomEvent === 'function') {
+      return new CustomEvent(event, params)
+    } else {
+      const evt = document.createEvent('CustomEvent')
```

Comment:
Can you name this variable something like `customEvent`?

Reply:
`customEvent` makes more sense to me since it's an object, not a method. Perhaps `customEventFactory` is appropriate here?

Reply:
Done!